### PR TITLE
db/datagrid-height

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed the double scroll issue for `DataGrid` in the `climate` demo.

### CODE REVIEW

File: samples/dashboards/demos/climate/demo.css [hunk 1]
```diff
@@ -267,6 +267,10 @@
     overflow: auto;
 }
 
+#selection-grid .hc-dg-outer-container {
+    height: 330px !important;
```
Comment (pawellysy) [inline samples/dashboards/demos/climate/demo.css:271]: Can we avoid using `!important`?  Maybe the design of the DataGrid is wrong, to set inner style on the element?

Comment (pawellysy) [reply samples/dashboards/demos/climate/demo.css:271]: The size is set by dataGridComponent. maybe we should add some margin there?

**Review by pawellysy — CHANGES_REQUESTED** [review]

File: samples/dashboards/demos/climate/demo.css (hunk 1 above)
Comment (sebastianbochan) [reply samples/dashboards/demos/climate/demo.css:271]: Maybe we should set this value from the JS?

**Review by bre1470 — APPROVED** [review]
Looks good. 👍

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**